contents/spotlight/startup-langfuse.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "How Langfuse pivoted and raised $4M after leaving Y Combinator "
+date: 2024-01-16
+author: ["joe-martin"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/startup_beforesunset.png
+featuredImageType: full
+category: Startups
+---
+
+> Each month, we speak to a team enrolled in [PostHog for Startups](/startups) about the way they work. This time, we spoke to [Langfuse](https://langfuse.com/), an open source observability platform for LLMs, about their Y Combinator experience.  
+
+For the team at Langfuse, the past 12 months have been a blur of activity and highlights. After completing Y Combinator’s W23 batch they’ve earned over 1.5k [GitHub stars](/spotlight/startup-documenso), won Product of the Day on [Product Hunt](/spotlight/startup-before-sunset-ai) and raised a $4M seed round backed by Lightspeed Venture Partners. 
+
+“We’ve been very lucky,” says Langfuse co-founder Marc Klingen. “Not only have we had the support and encouragement of [Y Combinator](/customers/ycombinator), but we’ve also benefited from the surging popularity of the LLM community too. There are so many teams solving problems with AI at the moment — it’s been a big plus for us.”
+
+Langfuses’ success is attributable to more than just good luck and the raise of AI, however. They’ve also succeeded by taking advantage of the opportunities available to them and applying some hard-learned lessons from their time in Y Combinator...
```

Comment:
Langfuse's instead of Langfuses’
rise of AI instead of raise of AI

Reply:
```suggestion
Langfuse's success is attributable to more than just good luck and the rise of AI, however. They’ve also succeeded by taking advantage of the opportunities available to them and applying some hard-learned lessons from their time in Y Combinator...
```